.github/workflows/merge.yml

```diff
@@ -229,84 +194,35 @@ jobs:
           SN_LOG: "all"
         timeout-minutes: 2
 
-      - name: Kill all nodes (unix)
-        shell: bash
-        timeout-minutes: 1
-        continue-on-error: true
-        run: |
-          pkill safenode
-          echo "$(pgrep safenode | wc -l) nodes still running"
-        if: failure() && matrix.os != 'windows-latest'
-
-      - name: Kill all nodes (windows)
-        shell: pwsh
-        timeout-minutes: 1
-        continue-on-error: true
-        run: Get-Process safenode | Stop-Process -Force
-        if: failure() && matrix.os == 'windows-latest'
-
-      - name: Tar log files (unix)
-        shell: bash
-        continue-on-error: true
-        run: |
-          find "${{ matrix.node_data_path }}" -iname '*.log*' | tar -zcvf log_files.tar.gz --files-from -
-        if: failure() && matrix.os != 'windows-latest'
-
-      - name: Tar log files (windows)
-        shell: pwsh
-        continue-on-error: true
-        run: |
-          Get-ChildItem -Path "${{ matrix.node_data_path }}" -Filter "*.log*" -Recurse -File | `
-            ForEach-Object {& 'tar' -czvf 'log_files.tar.gz' $_.FullName}
-        if: failure() && matrix.os == 'windows-latest'
-
-      - name: Upload Node Logs
-        uses: actions/upload-artifact@main
+      - name: Stop the local network and upload logs
+        if: always()
```

Comment:
this will upload log zips for successfully runs as well.
could be bit waste of resources.
Shall it got split into two and make the `upload logs` one only execute in failure?